### PR TITLE
Vds dev merging

### PR DESCRIPTION
**Pull request by mandolout**
Includes the most updated versions of the following servers:

serial_server.py - added search for COM Ports up to 48 (max was 20 previously)

quad_ad5780.py - Fixed output clogging. Made voltage signals send on reset. Added ramp signals. Fixed a typo.

ad5764_dcbox.py - Removed channel specific signals. Updated signal data for consistency. 

dac_adc.py - Added signals. Now enforces appropriate bounds for set_conversiontime. Updated device naming for consistency. Added send_read_requests setting. 

ad5764_acbox.py - Fixed boards not auto-updating when a voltage (or phase or frequency)  was changed.

### CODE REVIEW

File: quad_ad5780.py [hunk 1]
```diff
@@ -75,9 +75,18 @@ def shutdown(self):
     def do_init(self):
         yield self.packet().write("INITIALIZE\r").send()
         p=self.packet()
-        p.read_line()
         ans = yield p.send()
-        returnValue(ans.read_line)
+
+        done = False # wait for "INITIALIZATION COMPLETE"
+        while not done:
```
Comment (ckometter) [inline quad_ad5780.py]: Have you been having trouble initializing the quad?  `p.send()` is a blocking call; it'll wait until it gets a response or until the quad timeouts. 
`while not done:`  If the quad never initializes , it'll go into an infinite loop.

**Comment by mandolout** [comment]
Approved

**Comment by mandolout** [comment]
Aproved

**Comment by mandolout** [comment]
Approved

**Comment by ckometter** [comment]
Approved

**Pull request merged**